### PR TITLE
feat: add an option to enable/disable splunk

### PR DESCRIPTION
**Pull request by omrozowicz-splunk**

### CODE REVIEW

File: charts/splunk-connect-for-snmp/templates/common/sim-config.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+{{- if .Values.sim.enabled }}
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ include "splunk-connect-for-snmp.name" . }}-otel
+  name: {{ include "splunk-connect-for-snmp.name" . }}-sim
   namespace: {{ .Release.Namespace | quote }}
 data:
-  otel-collector-config.yaml: |
+  sim-collector-config.yaml: |
```
Comment (weliasz) [inline charts/splunk-connect-for-snmp/templates/common/sim-config.yaml:8]: This one is actually OTEL config

File: docs/gettingstarted/sc4snmp-installation.md [hunk 2]
```diff
@@ -112,15 +113,15 @@ Other variables to update in case you want to:
...
+required to change properties: `enabled` to `true` in `sim` section.
+Additionally connection parameter for Splunk Infrastructure Monitoring need to be set in `sim` section:
 
 | variable | description | default |
 | --- | --- | --- |
 |signalfxToken | SIM token which can be use for ingesting date vi API | not set|
 |signalfxRealm | Real of SIM | not set |
 
-For more detail please check [Otel Configuration](../configuration/otel-configuration.md)
+For more detail please check [Splunk SIM Configuration](../configuration/sim-configuration.md)
```
Comment (weliasz) [inline docs/gettingstarted/sc4snmp-installation.md:124]: Maybe just SIM

File: charts/splunk-connect-for-snmp/templates/common/sim-config.yaml (hunk 1 above)
Comment (omrozowicz-splunk) [reply charts/splunk-connect-for-snmp/templates/common/sim-config.yaml]: refactored

File: docs/gettingstarted/sc4snmp-installation.md (hunk 2 above)
Comment (omrozowicz-splunk) [reply docs/gettingstarted/sc4snmp-installation.md]: Changed

**Review by weliasz — APPROVED** [review]

**Pull request merged**